src_py/apiServer/experiment_flow.py

```diff
@@ -57,12 +58,12 @@ def get_events_sync(self):
     def get_csv_dataset(self):
         return self.csv_dataset
 
-    def generate_stats(self, experiment_phase = None) -> Stats:
+    def generate_stats(self, experiment_phase = None) -> StatsTiles:
```

Comment:
Please change it back to Stats object.
If you want to generate StatsTiles object please define new function that generate it.